recipes/openmpi/meta.yaml

```diff
@@ -0,0 +1,74 @@
+{% set version = "2.0.0" %}
+{% set major = ".".join(version.split(".")[:2]) %}
+
+package:
+    name: openmpi
+    version: {{ version }}
+
+source:
+    fn: openmpi-{{ version }}.tar.bz2
+    url: https://www.open-mpi.org/software/ompi/v{{ major }}/downloads/openmpi-{{ version }}.tar.bz2
+    sha1: a34f07a0e94af4ccd8080d74827befd61824c6c3
+
+build:
+    number: 0
+    skip: True  # [win]
+    binary_relocation: true
+    detect_binary_files_with_prefix: true
+
+requirements:
+    build:
+        - perl
+        - gcc  # [linux]
```

Comment:
I could not get `Fortran` support to work on `OS X` yet (I don't have a Mac so testing is quite slow b/c I rely only on `Travis-CI`.) Without `Fortran` there is no need for the `gcc` package.

PS: In the future I hope `conda-forge` packages its own `gcc` and use it everywhere to avoid these inconsistencies. But that is something for the long term...